surveys/2024-annual-survey/questions.md

```diff
@@ -414,11 +382,33 @@ Type: select all that apply (optional)
 - Achieving structured concurrency with async code
 - Writing executor-agnostic async code
 - Borrow checker not allowing valid code
+
+- (From here the additional one)
+- Runtime performance
+- Compile times
+- Binary size (of the final compiled artifact)
+- Memory usage (example: how much RAM rustc uses when compiling)
+- Disk space usage (example: the size of target folder)
+- Bugs in the compiler (that is, ICEs a.k.a. internal compiler errors, miscompilations, etc.)
+- Opaque / unclear compiler error messages
+- IDE experience
```

Comment:
IDE is a bit vague (I use emacs, you use vstudio). If this question means rust analyzer maybe explicit it?

Reply:
I would reword to something like "Insufficient IDE support" or "Rust IDE support is not good enough or is too slow", or something like that.

Reply:
I would keep the wording neutral because the quality assessment (good/bad) is up to the surveyer.

I was just wondering if it makes sense to coalesce all IDEs together, each of them has a completely different story and lifecycle. Maybe I use X and it's a stellar experience and you use Y and it's bad, it's apple and oranges and these differences are lost here.

Well, let's leave it like that, there's a specific question about "which IDE do you use" :man_shrugging: